consul/tests/test.bal

```diff
@@ -54,7 +54,7 @@ function testGetService() {
         io:println(<string>serviceDetails.detail().message);
         test:assertFail(msg = <string>serviceDetails.detail().message);
     } else {
-        test:assertNotEquals(serviceDetails, null, msg = "Failed to call getService()");
+        test:assertTrue(msg = "Failed to call getService()", serviceDetails.length() > 0);
```

Comment:
Can we swap the arguments please? Let's make the failure message the second argument?